e2e/specs/site.go

```diff
@@ -40,6 +50,27 @@ func VerifySiteUpdate(f *framework.T) {
 
 			Expect(newVersions).NotTo(BeEmpty(), "should have at least 1 new version available")
 			appPage.UpdateApp(newVersions[0])
```

Comment:
I know it's not your code - but since `GetNewVersions` does not return an error and does not validate if it obtained at least a single version, can we fix this to avoid potential out-of-bounds error?